ark/utils/metacluster_remap_gui/metaclusterdata.py

```diff
@@ -0,0 +1,128 @@
+from pathlib import Path
+
+import pandas as pd
+from scipy.cluster.hierarchy import ward
+from sklearn.metrics.pairwise import cosine_similarity
+
+
+def metaclusterdata_from_files(cluster_filepath, pixelcount_filepath, metacluster_header='metacluster'):  # noqa
+    clusters = pd.read_csv(cluster_filepath)
+    assert 'cluster' in clusters.columns, "cluster csv must include column named \"cluster\""
+    assert metacluster_header in clusters.columns, "cluster csv must include column named \"metacluster\", alternately specify the metacluster indexs using keyword `metacluster_index`"  # noqa
+    clusters = clusters.rename(columns={metacluster_header: 'metacluster'})
+    pixelcounts = pd.read_csv(pixelcount_filepath)
+    assert 'cluster' in pixelcounts.columns, "pixelcounts csv must include column named \"cluster\""  # noqa
```

Comment:
`assert` statements should be reserved for tests only, please `Raise` an error instead for input validation. 

We have a couple helper functions for input validation here: https://github.com/angelolab/ark-analysis/blob/31cc477932b28eba05c3329c807ac2237cf0b143/ark/utils/misc_utils.py#L120

For an example of how they're used see here, or just click on the def in github and go to `references` https://github.com/angelolab/ark-analysis/blob/31cc477932b28eba05c3329c807ac2237cf0b143/ark/mibi/tiling_utils.py#L33